WSSiOS/WSSiOS/Source/Presentation/MyPage/Profile/ProfileViewModel/MyPageEditProfileViewModel.swift

```diff
@@ -87,13 +96,37 @@ final class MyPageEditProfileViewModel: ViewModelType {
     func transform(from input: Input, disposeBag: DisposeBag) -> Output {
         let output = Output()
         
-        // 데이터 초기 설정
-        self.userNickname.accept(self.profileData.nickname)
-        self.userIntro.accept(self.profileData.intro)
-        self.userGenre.accept(self.profileData.genrePreferences)
-        self.userImage.accept(self.profileData.avatarImage)
-        
-        output.bindProfileData.accept(self.profileData)
+        // 진입 경로 별 데이터 바인딩 분리
+        if entryType == .myPage {
+            guard let profileData = self.profileData else { fatalError("프로필 데이터가 전달되지 않음") }
+            
+            self.userNickname.accept(profileData.nickname)
+            self.userIntro.accept(profileData.intro)
+            self.userGenre.accept(profileData.genrePreferences)
+            self.userImage.accept(profileData.avatarImage)
+            
+            output.bindProfileData.accept(profileData)
+            
+        } else if entryType == .home {
+            Observable.just(())
+                .flatMapLatest { [weak self] _ -> Observable<MyProfileResult> in
+                    guard let self = self else { return Observable.empty() }
+                    return self.getProfileData()
+                }
```

Comment:
W1;
이미 self.getProfileData 가 Observable을 뱉고 있어서 이렇게 안해줘도 될 것 같아유
```suggestion
            self.getProfileData()
```

Reply:
우왕 최고!